compiler/rustc_middle/src/mir/interpret/allocation.rs

```diff
@@ -629,15 +629,19 @@ impl InitMask {
         Size::from_bytes(block * InitMask::BLOCK_SIZE + bit)
     }
 
-    pub fn new(size: Size, state: bool) -> Self {
+    pub fn new_init(size: Size) -> Self {
         let mut m = InitMask { blocks: vec![], len: Size::ZERO };
-        m.grow(size, state);
+        m.grow(size, true);
         m
```

Comment:
We could keep the old API by using `if state { ... } else { ... }` -- that seems cleaner (not bothering the users with a somewhat cumbersome API)?